### ISSUE TITLE
Base Component BaseDatePick Freezing when selecting date in 4.0.1

### ISSUE THREAD

**Issue by betinhosampler** [bytefury/crater#321]
I don't know if it's a bug, but in all date components, when selecting a date, the screen freezes as if it were in an infinite loop, commenting out the line in the component's "onDateChange" method returns the normal behavior, however, the selected one date is not passed to the vue.

  methods: {
    onDateChange(date) {
      this.$emit('input', date)
      this.$emit('change', date)
    },
  },

#      this.$emit('input', date)
#      this.$emit('change', date)

I'm using "yarn install" and yarn watch
everything compiles normal, but after upgrading to version 4.0.0 and 4.0.1 the screens f the budget, invoice, expenses etc. nor open, only after commenting the lines above that I can see the screens, but without success to pass the selected dates.

Sorry for the english, google translator

- Crater version: 4.0.1
- PHP version: 7.0
- Database type and version: mariaDB

- OS:  [Windows 10]
- Browser: [chrome]

**Comment by mohitpanjwani** [comment]
Hey, @betinhosampler Thanks for letting us know about the issue. 

We've applied a fix on the demo site based on the feedback.. can you try creating an invoice and let me know it still crashes for you?

https://demo.craterapp.com/

----

Also, this is a duplicate of #316

**Comment by betinhosampler** [comment]
Olá, @mohitpanjwani, fiz o teste no demo como pediu, e ainda sem sucesso, nenhum erro no console, mas a tela simplesmente congela ainda, a listagem abre normal, mas na criação de nova fatura nao vai.

**Comment by mohitpanjwani** [comment]
Thanks for testing @betinhosampler. 

I've removed a watcher from BaseDatepicker and pushed the changes on demo. 

Can you refresh the page and try again now? Hopefully it should work now 🤞

**Comment by betinhosampler** [comment]
Bravo @mohitpanjwani, agora sim tudo certo, funcionou perfeitamente como o esperado, maravilha

**Comment by mohitpanjwani** [comment]
Awesome! Thanks for letting me know. 

Btw we've released a new update which includes this fix. 

Please update from Settings -> Update App Screen. 

Release: https://github.com/bytefury/crater/releases/tag/4.0.2
